tests/common/snappi_tests/traffic_generation.py

```diff
@@ -690,3 +710,80 @@ def verify_egress_queue_frame_count(duthost,
                 total_egress_packets, _ = get_egress_queue_count(duthost, peer_port, prios[prio])
                 pytest_assert(total_egress_packets == test_tx_frames[prio],
                               "Queue counters should increment for invalid PFC pause frames")
+
+
```

Comment:
please add comments on how the verification is done.

Reply:
the verification is done by checking the loss% in the flow stats of testflow and background flow. Testflow should have close to 16% loss and no loss to be observed on the background flow